src/utils/fileHasChanged.js

```diff
@@ -0,0 +1,140 @@
+              objectToCheck.content.interface.type !==
+              objectOfRef.content.interface.type
+            ) {
+              resultSet.content.interface.type = true
+            }
+          }
+          //Vérifie si la donnée availableLabels a changé
+          if (
+            typeof objectToCheck.content.interface.availableLabels !==
+            "undefined"
+          ) {
+            if (
+              typeof objectOfRef.content.interface.availableLabels ===
+              "undefined"
+            ) {
+              resultSet.content.interface.availableLabels = true
+            } else if (
+              objectToCheck.content.interface.availableLabels !==
+              objectOfRef.content.interface.availableLabels
+            ) {
+              resultSet.content.interface.availableLabels = true
+            }
+          }
+          //Vérifie si la donnée regionTypesAllowed a changé
+          if (
+            typeof objectToCheck.content.interface.regionTypesAllowed !==
+            "undefined"
+          ) {
+            if (
+              typeof objectOfRef.content.interface.regionTypesAllowed ===
+              "undefined"
+            ) {
+              resultSet.content.interface.regionTypesAllowed = true
+            } else if (
+              objectToCheck.content.interface.regionTypesAllowed !==
+              objectOfRef.content.interface.regionTypesAllowed
+            ) {
+              resultSet.content.interface.regionTypesAllowed = true
+            }
+          }
+        }
+      }
+      //Vérifie si la donnée taskData a changé
+      if (typeof objectToCheck.content.taskData !== "undefined") {
+        if (typeof objectOfRef.content.taskData === "undefined") {
+          resultSet.content.taskData = true
+        } else if (
+          objectToCheck.content.taskData !== objectOfRef.content.taskData
+        ) {
+          resultSet.content.taskData = true
+        }
+      }
+      //Vérifie si la donnée taskOutput a changé
+      if (typeof objectToCheck.content.taskOutput !== "undefined") {
+        if (typeof objectOfRef.content.taskOutput === "undefined") {
+          resultSet.content.taskOutput = true
+        } else if (
+          objectToCheck.content.taskOutput !== objectOfRef.content.taskOutput
+        ) {
+          resultSet.content.taskOutput = true
+        }
+      }
+    }
+  }
+
+  // Vérifie si la donnée mode existe et est changé
+  if (typeof objectToCheck.mode !== "undefined") {
+    if (typeof objectOfRef.mode === "undefined") resultSet.mode = true
+    else if (objectToCheck.mode !== objectOfRef.mode) resultSet.mode = true
+  }
+
+  // Vérifie si la donnée fileName existe et est changé
+  if (typeof objectToCheck.fileName !== "undefined") {
+    if (typeof objectOfRef.fileName === "undefined") resultSet.fileName = true
+    else if (objectToCheck.fileName !== objectOfRef.fileName)
+      resultSet.fileName = true
+  }
+  //Comportement par défaut
+  return resultSet
+}
```

Comment:
don't forget about poor mono-lingual people like me :)

Reply:
I will change the comment in english

Reply:
cheers :)